erpnext/patches/v13_0/make_homepage_products_website_items.py

```
@@ -13,4 +13,7 @@ def execute():
 		row.item_code = web_item
 
 	homepage.flags.ignore_mandatory = True
-	homepage.save()
\ No newline at end of file
+	try:
+		homepage.save()
+	except frappe.exceptions.LinkValidationError:
```

ignore links instead? ignoring exception will commit a potentially partial change. 

`homepage.flags.ingore_links = True`